http.go

```diff
@@ -1652,6 +1654,11 @@ func appendBodyFixedSize(r *bufio.Reader, dst []byte, n int) ([]byte, error) {
 	}
 }
 
+// ErrBrokenChunks is returned when server got broken chunked body (Transfer-Encoding: chunked).
+type ErrBrokenChunks struct {
```

Comment:
Sorry for being pedantic but can you change this to `ErrBrokenChunk`. The error is only about one chunk being broken, not multiple.